externals/global_params.php

```diff
@@ -82,6 +82,6 @@
      $URL_WIKI="https://wiki.gnucash.org/wiki/";
 
      // some defaults for metatags in case the page does not set them
-     $default_title= T_("Free Accounting Software");
-     $default_description= T_("<span class=\"gnucash\">GnuCash</span> is personal and small-business financial-accounting software, freely licensed under the <a href=\"https://www.gnu.org/\">GNU</a> GPL and available for GNU/Linux, BSD, Solaris, Mac OS X and Microsoft Windows.");
+     $default_title= T_("Copylefted Libre Software");
+     $default_description= T_("<span class=\"gnucash\">GnuCash</span> is financial-accounting software for personal and small-business use. Freely licensed <a href=\"https://www.gnu.org/\">GPLv2+</a>. Available for GNU/Linux, BSD, Solaris, macOS and Windows.");
```

Comment:
I was working on this, too.
`GPLv2+`: +1

1. drop Solaris
2. insert double entry accounting
3. See https://wiki.gnucash.org/wiki/Website_Maintenance#Metatags

Finaly I was replacing the markup parts by parameters.

Reply:
"Financial accounting" shouldn't be hyphenated.

If you're going to change the dependent clause into a separate sentence then it needs a verb, e.g. `It is freely licensed...`. "Freely licensed" doesn't mean the same as "Licensed as Free". 

The URI should match the term, so it should be `https://www.gnu.org/licenses/old-licenses/gpl-2.0.en.html`, and "GPLV2 or later" is better than GPL+.

"Windows" is not correct. "Microsoft Windows" is.

Reply:
One more thing: The commit summary is wrong. This changes mostly correct wording for other mostly correct wording. The new wording may or may not be more easily understandable to people for whom English is not their native language, but you're not correcting anything.

Reply:
All yes, except it is "GPLv2+" (now), not "GPL+", does your point still stand?
How is "Windows" not correct. Windows.com uses "Windows".

Reply:
"and later" is less ambiguous than "+". 

Windows.com redirects to Microsoft.com.

Microsoft has agressively asserted for years that it owns the word Windows as a trademark, but [got slapped down in court](https://en.wikipedia.org/wiki/Microsoft_Corp._v._Lindows.com,_Inc.) and settled. That hasn't stopped them from continuing to threaten. [Wikipedia's lead article on the OS](https://en.wikipedia.org/wiki/Microsoft_Windows) leads off with "Microsoft Windows, commonly referred to as Windows...".

Reply:
Microsoft.com is the older domain?

What am I missing there? GnuCash could get sued for using "Windows" as a proper noun when Microsoft does so without specifying trademark? GnuCash in so doing isn't infringing on the likeness of the trademark, or even in the same business. It is what Windows wants to do, so why shouldn't it want GnuCash to do it? Is there an ethical consideration to be made here?

It isn't actually "and later", it is "or later", at-least in the standard formatting of the license name https://spdx.org/licenses/GPL-2.0-or-later.html
Specifically "or (at your option) any later version" is the nomenclature of https://www.gnu.org/licenses/old-licenses/gpl-2.0.en.html

Wikipedia uses "+" I seem to remember https://en.wikipedia.org/wiki/LibreOffice
Could be wrong, and it is only the chosen displayed name in this case.

Reply:
> Microsoft.com is the older domain?

Probably, because Microsoft would have been in Arpanet before there would have been a reason to have a windows.com. That would have come along in the mid-to-late 90s when advertising on the WWW became a thing. Not sure that that's relevant.

You're right, it should be "or later", not "and later".

No, I don't think that Microsoft is going to pester us about using the word Windows to describe their OS. That's what they want. I prefer to make clear that there are other windowing operating systems by not using Windows by itself. 

But your comment about SEO led me to dig a bit deeper. I tried DuckDuckGo (because Google firmly associates my IP with GnuCash) searches for "free accounting software", "libre accounting software", "copyleft accounting software", and "gpl accounting software". www.gnucash.org was on the first page only for the last search. Google searches for the same terms present GnuCash on the same page for each.

Google rearranges the title (which gets " | GnuCash" appended in header.phtml) to GnuCash: Free Accounting Software while DDG leaves it as Free Accounting Software | GnuCash. The default_description is placed both in the meta description and the first paragraph of index.phtml so it's not possible to tell which Google presents under the title, but DDG concats it with the first sentence of the second para ("Designed to be easy to use..."). Since more than 90% of web searches are on Google what they show is most important and they chop the current description after Mac OS. That means we have about 130 characters to sell searchers on clicking through so we should focus it on GnuCash features and (aside from it being free and cross-platform) push details like the laundry list of supported OSes to after the 130 characters. Since that laundry list will be used for harvesting search terms we absolutely want to say Apple macOS and Microsoft Windows so that the company names are included as search terms. Licensing details including terms like copyleft, free, and libre should also go in that part of the description. 

The meta title is used as the tab title too, progressively truncated depending on tab size. With the current title and around 10 tabs it's just the favicon and "Free". ISTM we want that to be GnuCash so that should be the first word rather than the last and "GnuCash Accounting" makes more sense than "GnuCash Free", never mind "GnuCash Copyleft".